### PR TITLE
changing names of methods and classes

### PR DESCRIPTION
**Pull request by SabBabQab**
Please merge

### CODE REVIEW

File: MadameMaths/CheckUserAnswer.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿using System;
 namespace MadameMaths
 {
-    public static class GiveSolution
+    public static class CheckUserAnswer
```
Comment (CrazyDanyal1414) [inline MadameMaths/CheckUserAnswer.cs:4]: @SabBabQab Change to `UserAnswer`

File: MadameMaths/CheckUserAnswer.cs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
     {
-        public static int WrongOrRight(double correctAnswer, double danswer, int score)
+        public static int CheckingUserAnswer(double correctAnswer, double finalAnswer, int score)
```
Comment (CrazyDanyal1414) [inline MadameMaths/CheckUserAnswer.cs:6]: @SabBabQab You already have another method called CheckUserAnswer. This one can be removed, no?

File: MadameMaths/CorrectAnswer.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+namespace MadameMaths
+{
+    public static class CorrectAnswer
+    {
+        public static double GetCorrectAnswer(double randomNumber1, string operatorSymbol, double randomNumber2)
```
Comment (DanCodes1414) [inline MadameMaths/CorrectAnswer.cs]: This method can be put inside the `UserAnswer` class

File: MadameMaths/GamePlay.cs [hunk 4]
```diff
@@ -0,0 +1,50 @@
+﻿using System;
+namespace MadameMaths
+{
+    public static class GamePlay
+    {
+        public static int PlayingGameBasedOnDifficulty(string difficulty, int numberOfQuestions)
```
Comment (DanCodes1414) [inline MadameMaths/GamePlay.cs]: Can be renamed to `PlayGameBasedOnDifficulty`

**Review by DanCodes1414 — CHANGES_REQUESTED** [review]
Please make these changes

**Review by DanCodes1414 — APPROVED** [review]
Well done!

**Pull request merged**